.gitignore

```diff
@@ -42,6 +42,7 @@ node_modules
 /telegraf-docs/
 /typings/
 tsconfig.tsbuildinfo
+/bin/telegraf.mjs
```

Comment:
Why bother? https://github.com/telegraf/telegraf/issues/1384

Reply:
I found new purpose for the bin:

https://github.com/feathers-studio/telegraf-docs/blob/2b4eb11c1aeff01e6f9f22bad99843eba6e60189/examples/functions/aws-lambda/package.json#L11

```shell
telegraf -m setWebhook -D '{ "url": $WEBHOOK_URL }'
```

Reply:
To be used manually after publishing to a FaaS, or in CI; of course it could be done with a HTTP URL, it's just a nicer way and opens up more scripting for almost no extra cost.

Reply:
Still, I'd rather you revitalized `micro-bot` to avoid the dependencies

Reply:
`@telegraf/cli` would be a better name

Reply:
Yep, I thought about it. Since it's a breaking change, we'll release for now, and move it to `@telegraf/bin` in v5. The deps incurred are tiny anyway, node-fetch is the largest dependency, and by a lot.